substra/sdk/client.py

```diff
@@ -97,24 +95,34 @@ def __init__(self, config_path=None, profile_name=None, user_path=None,
         # set current logged user if exists
         self.set_user()
 
+        if token:
+            self.set_token(token)
+
     @logit
-    def login(self):
+    def login(self, username, password):
         """Login.
 
         Allow to login to a remote server.
 
-        After setting your configuration with `substra config` using `-u` and `-p`
-        Launch `substra login`
-        You will get a token which will be stored by default in `~/.substra-user`
+        After setting your configuration with `substra config`, launch `substra login`.
+        You will be prompted for your username and password and get a token which will be
+        stored by default in `~/.substra-user`
         You can change that thanks to the --user option (works like the --profile option)
```

Comment:
Not related directly to this PR, but this doctring seems related to the CLI (and not the SDK).

Reply:
Agreed, I'll change it.